### PR TITLE
isHide to isInvisible

### LINKED ISSUE #413: Make view binding isHide to isInvisible
## Kind (Required)

- Improvement

 ## Overview (Required)

There is binding. But when `show` is true, the view becomes visible 😭 
Can you use AndroidX isInvisible in that method?
And Can you change Hide to isInvisible?

```
@BindingAdapter("isHide")
fun View.showHide(show: Boolean) {
    visibility = if (show) View.VISIBLE else View.INVISIBLE
}
```

```
inline var View.isInvisible: Boolean
    get() = visibility == View.INVISIBLE
    set(value) {
        visibility = if (value) View.INVISIBLE else View.VISIBLE
    }
```

 ## Links	

-

### PR DESCRIPTION
**Pull request by sasakitomohiro**
## Issue
- close #413 

## Overview (Required)
- Make view binding isHide to isInvisible

## Links
-

## Screenshot
I did not modified.
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by roana0229** [comment]
👀

File: feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
             android:text="@string/start_filter"
             android:textAppearance="@style/TextAppearance.DroidKaigi.Body2"
             app:icon="@drawable/ic_filter_list_black_24dp"
-            app:isHide="@{(!isEmptySessions || isFiltered) &amp;&amp; !isCollapsed}"
+            app:isInvisible="@{(isEmptySessions || !isFiltered) &amp;&amp; isCollapsed}"
```
Comment (roana0229) [inline feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml]: Could you please change `inverted value` to `suggestion value` ?

```suggestion
            app:isInvisible="@{(isEmptySessions &amp;&amp; !isFiltered) || isCollapsed}"
```

File: corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/binding/Bindings.kt [hunk 2]
```diff
@@ -11,9 +12,9 @@ fun View.showGone(show: Boolean) {
     isVisible = show
 }
 
-@BindingAdapter("isHide")
+@BindingAdapter("isInvisible")
 fun View.showHide(show: Boolean) {
-    visibility = if (show) View.VISIBLE else View.INVISIBLE
+    isInvisible = show
```
[the remark this answers is not in the document]
Comment (sasakitomohiro) [reply corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/binding/Bindings.kt]: I agree with your idea!
I fix it now👍

File: feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml (hunk 1 above)
Comment (sasakitomohiro) [reply feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml]: You are right! I fix it!

File: corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/binding/Bindings.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (sasakitomohiro) [reply corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/binding/Bindings.kt]: I fixed it🙆‍♂️

File: feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml (hunk 1 above)
Comment (sasakitomohiro) [reply feature/session/src/main/res/layout/fragment_bottom_sheet_favorite_session.xml]: I fixed it🙆‍♂️

**Comment by roana0229** [comment]
Thanks for your fixes!
LGTM 👍

**Pull request merged**